Sanity-Check.md

```diff
@@ -1,3 +1,75 @@
+# Sanity Checking VS Code
+### Jump to a section:
+[About](#about) \
+[Getting Started](#getting-started) \
+[Steps to Test Per Build Type](#steps-to-test-per-build-type) 
+
+
+---
+---
+## About
+
+#### What is sanity testing and why do we do it?
+Sanity testing is a process where we confirm our vscode installers work and that the final build can run. We sanity test to make sure the builds we ship work on the most basic level and there isn’t an immediate breaking change on VS Code. It also allows us to have human eyes on the actual build we ship and avoids the use of potentially flakey tests in the final stages of a release. (generally just make sure we don’t embarrass ourselves by shipping something that doesn’t even start).
+
+#### What to look for when you sanity test?
+
+We are looking for errors, failures, or anything else that is not desired behavior. Examples are things not loading or errors when trying to open the build in the first place.
+
+#### What does it mean to sanity test and what are some things to try during sanity testing?
+
+- On the most basic level:  install, launch, then open a file or the about dialog
+- Other options to test a bit more:
+    - Click through all the items on the sidebar
+    - Try some command from the command palette
+    - Open / click around in the terminal
+
+---
+---
+## Getting Started
+
+When the step of sanity testing comes in endgame, sections are assigned based on device type. The devices tested are as follows with the compatible testing machine:
+
+| OS to Test      | Possible Testing Machine for Given OS |
```

Comment:
Windows ARM? only alternate is apple silicon AFAIK

Reply:
and Linux. Is there a reason this doesn't just match the list of OS's to test below?

Reply:
I'm a bit confused on the headings... it should read like this:
> If you have COLUMN 1, you can test COLUMN 2

Maybe 
```suggestion
| OS you are running | Additional builds you are able to test |
```

Reply:
ooooh, yeah, I had it backwards